app/javascript/api/listingApiService.ts

```diff
@@ -19,8 +19,15 @@ type ListingPreferencesResponse = { preferences: RailsListingPreference[] }
 type ListingUnitsResponse = { units: RailsUnit[] }
 type ListingAmiChartsResponse = { ami: RailsAmiChart[] }
 
-export const getListing = async (listingId?: string): Promise<RailsListing> =>
-  get<ListingsResponse>(listing(listingId)).then(({ data }) => data.listing)
+export const getListing = async (listingId?: string): Promise<RailsListing> => {
+  let force = "false"
+  if (window.location.search.includes("preview=true")) {
+    force = "true"
+  }
+  return get<ListingsResponse>(listing(listingId), { params: { force } }).then(
+    ({ data }) => data.listing
+  )
+}
```

Comment:
Did you check for other endpoints that need to support the flag?

I mentioned in a previous comment that we should check the Angular code to see which endpoints support the `preview=true` params, here are my results:
![Screenshot 2023-11-02 at 9 46 07 AM](https://github.com/SFDigitalServices/sf-dahlia-web/assets/133722877/9d11197c-7502-4bb9-b2fa-47fe35d9828c)

Reply:
This ticket is just for ListingDetails on React so that we can unblock the release. We can create a new ticket to handle any other endpoints or params

Reply:
> We can create a new ticket

When you create the ticket, please mark it as a higher priority. To my knowledge, the three other endpoints are all used on the Listing Details page.